ethportal-peertest/src/scenarios/find.rs

```diff
@@ -0,0 +1,84 @@
+use serde_json::{json, Value};
+
+use crate::{
+    jsonrpc::{make_ipc_request, JsonRpcRequest, HISTORY_CONTENT_VALUE},
+    Peertest, PeertestConfig,
+};
+use trin_core::jsonrpc::types::Params;
+
+pub fn test_trace_recursive_find_content(_peertest_config: PeertestConfig, peertest: &Peertest) {
```

Comment:
why the unused `_peertest_config` here and in other functions?

Reply:
I'm just following the pattern here. If you think it's worth going back and refactoring the pattern, let's do it in another pr/issue.